.schema/devbox.schema.json

```diff
@@ -0,0 +1,87 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "$id": "https://github.com/jetpack-io/devbox",
+    "title": "Devbox json definition",
+    "description": "Defines fields and acceptable values of devbox.json",
+    "type": "object",
+    "properties": {
+        "packages": {
+            "description": "Collection of packages to install",
+            "oneOf": [
+                {
+                    "type": "array",
+                    "items": {
+                        "description": "Name and version of each package in name@version format.",
+                        "type": "string"
+                    }
+                },
+                {
+                    "type": "object",
+                    "description": "Name of each package in {\"name\": \"version\"} format.",
+                    "patternProperties": {
+                        ".*": {
+                            "type": "string",
+                            "description": "Version of the package to install."
+                        }
+                    }
+                },
+                {
+                    "type": "object",
+                    "description": "Name of each package in {\"name\": {\"version\": \"1.2.3\"}} format.",
+                    "patternProperties": {
+                        ".*": {
+                            "type": "object",
+                            "description": "Version number of the specified package in {\"version\": \"1.2.3\"} format.",
+                            "properties": {
+                                "version": {
+                                    "type": "string",
+                                    "description": "Version of the package"
+                                },
+                                "platform": {},
+                                "excluded_platform": {},
```

Comment:
I think these should be `platforms` and `excluded_platforms` (note the `s` for plural)